launchdarkly-server-sdk/src/events/sender.rs

```diff
@@ -107,8 +107,8 @@ where
                 }
             };
 
-            for attempt in 0..2 {
-                if attempt == 1 {
+            for attempt in 1..=2 {
```

Comment:
I feel like it makes more sense to be 1-based in this situation, even though I typically prefer 0 based loops.